RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: enhancement
```

Comment:
The release type should be `patch`, `minor`, or `major`. This is used to bump the version number:

https://github.com/strawberry-graphql/strawberry/blob/772703c7c47173eeb9e4e8d4c43fccd28166d520/.github/release-check-action/release.py#L14-L17